### ISSUE TITLE
Book 05: Sentence 055

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#39]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/b18f2c97-0a63-45de-81e9-1ee821d38fe1)

```
# sent_id = Book_05_-_The_Omens.txt:55
# text = Ic chicome capitulo, vncan mjtoa: in tetzaujtl injc motetxauja, in jtechpa tochin: in jquac qujttaia in je vncalaquj ichan.
# text[norm] = Ic chicome capítulo, oncan mihtoa: in tetzahuitl inic motetzahuia, in itechpa tochin: in ihcuac quittaya in ye oncalaqui ichan.
# text[orig] = Ic·chicome·capitulo·,·vncan·mjtoa·:·in·tetzaujtl·injc·motetxauja·,·inj·techpa¶tochin·:·injquac·qujttaia·in·jevnca·laquj¶ichan·.
# text[osp] = Capitulo septimo, en que le tracta, del mal aguero que tomauan: quando vian algun conejo entrar en su casa.
# text[osp_norm] = Capítulo séptimo, en que le trata, del mal agüero que tomaban: cuando veían algún conejo entrar en su casa.
# references = A&D/170:4
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 62.50%
1	Ic	ic	ADV	_	_	3	amod	_	_
2	chicome	chicome	NUM	_	_	1	nummod	_	_
3	capitulo	capítulo	NOUN	_	State=Abs	6	obl	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	vncan	oncan	ADV	_	_	6	advmod	_	_
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
7	:	:	PUNCT	_	_	9	punct	_	_
8	in	in	DET|SCONJ	_	_	9	det	_	_
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	_	_	_
10	injc	inic	SCONJ	_	_	11	mark	_	_
11	motetxauja	tetzahuia	VERB	_	_	9	_	_	_
12	,	,	PUNCT	_	_	14	punct	_	_
13	in	in	DET|SCONJ	_	_	14	_	_	_
14	jtechpa	techpa	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	11	_	_	_
15	tochin	tochin	NOUN	_	State=Abs	14	nmod	_	_
16	:	:	PUNCT	_	_	19	punct	_	_
17	in	in	SCONJ	_	_	19	mark	_	_
18	jquac	ihcuac	ADV	_	_	19	mark	_	_
19	qujttaia	itta	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	11	_	_	_
20	in	in	SCONJ	_	_	22	mark	_	_
21	je	ye	ADV	_	_	22	advmod	_	_
22	vncalaquj	calaqui	VERB	_	Movement=And|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	19	_	_	_
23	ichan	chantli	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Pos	22	obl	_	_
24	.	.	PUNCT	_	_	6	punct	_	_
```

**Comment by ftyers** [comment]
* *in tetzahuitl inic motetzahuia*, see also #11.

**Comment by ftyers** [comment]
Updated:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/cb4eecbc-26f2-4b94-bb48-ac53d97d501f)

Questions:
- For the *n*th time, is *in tetzahuitl, inic motetzahuia* conjoined clauses?
- Is *in itechpa tochin* clausal or nominal, what does it modify?
- What does *in ihcuac quittaya* modify?

**Comment by Lguyogiro** [comment]
> For the nth time, is in tetzahuitl, inic motetzahuia conjoined clauses?

I think yes. I've been trying to figure out what is going on with `inic` here....page 94 in Lockhart (english edition, lesson 18) gives an example of how _inic_ is used as "how/that" after _ihtoa_. I think this is the most plausible interpretation semantically here of _inic_ because the other possible meanings (e.g. "since", "when", "so that", etc) don't seem to fit. So if that is the case, then we would probably want to say that _inic_ is subordinating the complement of _ihtoa_, thus making it conjoined with _in tetzahuitl_

**Comment by Lguyogiro** [comment]
> Is in itechpa tochin clausal or nominal, what does it modify?

I think it is clausal here: "it is about the rabbit" or "it deals with the rabbit". I think this should be `parataxis:shared` from _tetzahuitl_.

**Comment by Lguyogiro** [comment]
> What does in ihcuac quittaya modify?

this is kind of a weird one. I think our current analysis makes sense (parataxis from tetzahuitl). However I feel weird with this because of the coreference with _tochtli_. But floco doesn't care about my feelings....

**Comment by Lguyogiro** [comment]
actually, I guess it would be parataxis from _mijtoa_

**Comment by Lguyogiro** [comment]
Ok, actually I now see some examples where Lockhart translates _ic_ and _inic_ as "with which"....I believe this is also how Dibble and Anderson treat it in the sentence above (and others). Under that translation/analysis of the subordinator, I would think we would actually want something like `acl(tetzahuitl, motetzahuia)`. 

Thoughts?

**Comment by Lguyogiro** [comment]
regarding _in itechpa_ , since we are treating this as a clause, we probably want to select SCONJ and mark (same as in #33 )

**Comment by ftyers** [comment]
I think it depends on if we treat *in tetzahuitl* as clausal or nominal. If it's nominal then:
* *a*/`NOUN` + inic + *b*/`VERB` → `acl(a, b)`, `mark(b, inic)`

If it's verbal then it would be either:

* *a*/`VERB` + inic + *b*/`VERB` → `advcl(a, b)`, `mark(b, inic)`
* *a*/`VERB` + in ic + *b*/`VERB` → `conj(a, b)`, `mark(b, inic)`

The analysis with `conj` is basically that we imagine we have:

* *in tetzahuitl (catca), in ic motetzahuia* 

where this is two clauses introduced by *in*, "[that there was an omen] and [that how they considered it to be an omen]"

**Comment by Lguyogiro** [comment]
alternatively, we could say that _in tetzahuitl in ic motetzahuiaya_ is a nominal clause right? in which case the internal structure could still have `acl`?

**Comment by ftyers** [comment]
Like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/d9b99c49-0a2b-4aa3-91fe-867116c6b8b1)

**Comment by Lguyogiro** [comment]
yeah this is what I was thinking. `conj` would also be okay with me, it just depends on how much importance we put on the subordinator "inic" here...like every time we see "in NOUN inic VERB" would we also do `acl(NOUN, VERB)`? That would be okay with me.

If you are on board (and we don't see a million examples of this construction that complicate this analysis, then the tree above LGTM.
